crates/swc_ecma_transforms_module/src/common_js.rs

```diff
@@ -242,7 +254,6 @@ impl Fold for CommonJs {
                             init_export!(js_word!("default"))
                         }};
                         ($name:expr) => {{
-                            exports.push($name.clone());
```

Comment:
probably, all the exports should be collected in the preliminary pass and this call is unnecessary